Automattic/wp-calypso
# Reader Onboarding: add industry curated blogs

Related to Automattic/dotcom-forge#9439

## Proposed Changes

* Adds curated feeds/blogs for various industry sub-topics

(With Space, Business, and Remote Work followed)
| Before | After |
|-|-|
| ![image](https://github.com/user-attachments/assets/849fcd93-f531-4014-9fcc-d6ae021c6e18) | ![image](https://github.com/user-attachments/assets/dbfb374d-327b-4939-ba51-570aeee0e77e) |


## Why are these changes being made?


* The provide a better UX when suggesting sites users should subscribe to after they follow a tag

## Testing Instructions



* Subscribe to at least three of the curated industry tags: **"business"**, **"startups"**, **"finance"**, **"space"**, **"leadership"**, **"marketing"**, **"remote work"**, **"saas"**, or **"creator economy"**
* Navigate to `/read?flags=reader/onboarding` 
* Click "Discover and subscribe to sites you'll love"
* Check to see that recommendations make sense and you are able to subscribe

## Pre-merge Checklist
